container.go

```diff
@@ -160,7 +160,12 @@ func showContainerCheckpoint(checkpointDirectory string) error {
 			table.Append([]string{
 				data.Destination,
 				data.Type,
-				shortenPath(data.Source),
+				func() string {
+					if fullPaths {
+						return data.Source
+					}
+					return shortenPath(data.Source)
+				}(),
```

Comment:
Could we move the check for `fullPaths` before `table.Append`?
For example:
```Go
source := data.Source
if !fullPaths {
	source = shortenPath(source)
}
table.Append([]string{
	data.Destination,
	data.Type,
	source,
})
```

Reply:
So which one do we go ahead with? :confused:

Reply:
I'm sorry, I missed your comment. Let's leave as it is.